css/style.css

```diff
@@ -0,0 +1,461 @@
+}
+.nav-list li a:hover {
+  color: red;
+}
+
+.br-nav-item {
+  border-right: 1px solid #d9d9d9;
+}
+
+.nav-search {
+  top: 3px;
+  right: 105px;
+  width: 60px;
+  height: 49px;
+  font-size: 18px;
+  position: absolute;
+}
+
+.search-icon {
+  top: 20px;
+  clip: auto;
+  width: 20px;
+  right: 80px;
+  height: 20px;
+  font-size: 20px;
+  position: absolute;
+}
+
+.main {
+  padding: 20px 30px;
+}
+
+.block-tittle {
+  color: rgba(3, 3, 3, 0.4);
+  font-size: 16px;
+  padding-bottom: 11px;
+}
+
+.category {
+  padding: 8px 15px 9px;
+  background-color: rgba(255, 255, 255, 0.9);
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  z-index: 1;
+}
+.category a {
+  color: #f72210;
+  font-size: 14px;
+}
+
+.header-article {
+  position: relative;
+  margin-bottom: 10px;
+}
+.header-article a {
+  width: 100%;
+}
+
+.article h3 {
+  font-size: 1.32em;
+}
+
+.article-title a {
+  color: #000;
+}
+.article-title a:hover {
+  -webkit-text-decoration: underline red;
+  text-decoration: underline red;
+}
+
+.summary {
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.related {
+  border-top: 1px solid #d9d9d9;
+  border-bottom: 1px solid #d9d9d9;
+  margin-top: 16px;
+  padding: 6px 0;
+}
+.related a {
+  font-size: 16px;
+  line-height: 20px;
+  color: #000;
+  margin: 20px 0;
+}
+.related a:hover {
+  text-decoration: none;
+  color: red;
+}
+
+.ms-articles {
+  padding-top: 14px;
+  padding-bottom: 14px;
+  border-top: 1px solid #ddd;
+}
+
+.ms-titles {
+  font-size: 17px;
+  line-height: 20px;
+}
+
+.img-header {
+  width: 63px;
+  height: 63px;
+  background: url(../img/opinion-headshot-bg.png) center no-repeat #f72210;
+  -webkit-box-flex: 0;
+  -ms-flex: none;
+  flex: none;
+}
+
+.opinion-link {
+  font-size: 16px;
+  line-height: 20px;
+  color: #000;
+  margin: 20px 0;
+}
+.opinion-link:hover {
+  text-decoration: none;
+  color: red;
+}
+
+.opinion-by {
+  font-size: 14px;
+  line-height: 21px;
+  color: #555;
+  margin-top: 4px;
+}
+
+.border-li {
+  border-top: 1px solid #ddd;
+}
+
+.img-latest-news {
+  -webkit-box-flex: 0;
+  -ms-flex: none;
+  flex: none;
+  width: 135px;
+  margin-right: 15px;
+}
+
+.info-link {
+  font-size: 17px;
+  line-height: 20px;
+  color: #000;
+}
+.info-link:hover {
+  text-decoration: none;
+  color: red;
+}
+
+.newsletter {
+  max-width: 300px;
+  margin: 0 auto 30px;
+  border: 1px solid #e5e5e5;
+  background-color: #f4f5f6;
+  padding: 20px;
+}
+.newsletter a {
+  font-size: 15px;
+  color: #333;
+}
+
+.news-title {
+  font-size: 19px;
+}
+
+.news-icon {
+  -webkit-box-flex: 0;
+  -ms-flex: none;
+  flex: none;
+  width: 29px;
+  height: 28px;
+  background: url(https://s.newsweek.com/sites/www.newsweek.com/themes/newsweek/images/logo-n1.svg) center/contain no-repeat #f72210;
+  margin-right: 10px;
+}
+
+.news-btn {
+  padding: 10px 10px 10px 12px;
+  color: #fff;
+  margin-bottom: 15px;
+  background-color: #666;
+  font-size: 18px;
+}
+.news-btn::after {
+  content: "";
+  display: inline-block;
```

Comment:
`SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._